vaadin-combo-box-flow-parent/vaadin-combo-box-testbench/src/main/java/com/vaadin/flow/component/combobox/testbench/MultiSelectComboBoxElement.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.combobox.testbench;
+
+import com.vaadin.testbench.HasHelper;
+import com.vaadin.testbench.HasLabel;
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.testbench.elementsbase.Element;
+import org.openqa.selenium.By;
+
+import java.util.List;
+
+/**
+ * A TestBench element representing a
+ * <code>&lt;vaadin-multi-select-combo-box&gt;</code> element.
+ */
+@Element("vaadin-multi-select-combo-box")
+public class MultiSelectComboBoxElement extends TestBenchElement
+        implements HasLabel, HasHelper {
+
+    public String getInputElementValue() {
+        return this.getPropertyString("_inputElementValue");
+    }
+
+    /**
+     * Opens the popup, if it is not already open.
+     */
+    public void openPopup() {
+        setProperty("opened", true);
+    }
+
+    /**
+     * Close the popup, if it is open.
+     */
+    public void closePopup() {
+        setProperty("opened", false);
+    }
+
+    /**
+     * Checks whether the popup is open.
+     *
+     * @return <code>true</code> if the popup is open, <code>false</code>
+     *         otherwise
+     */
+    public boolean isPopupOpen() {
+        return getPropertyBoolean("opened");
+    }
+
+    /**
+     * Opens the popup, and gets the labels of the items that are currently
+     * loaded in the popup
+     *
+     * @return labels of the items that are loaded in the popup
+     */
+    @SuppressWarnings("unchecked")
+    public List<String> getOptions() {
+        openPopup();
+        waitForLoadingFinished();
+        //@formatter:off
+        String script =
+                "const comboBox=arguments[0];" +
+                "return comboBox.$.comboBox.filteredItems.map(item => item.label || '')";
+        //@formatter:on
+        return (List<String>) executeScript(script, this);
+    }
+
+    /**
+     * Attempts to select an item from the popup by matching the label. Throws
+     * an {@link IllegalArgumentException} if the popup does not contain an item
+     * with the specified label. Does nothing if the item is already selected.
+     *
+     * @param label
+     *            The label of the item to select from the popup
+     * @throws IllegalArgumentException
+     *             if there is no item with the specified label
+     */
+    public void selectByText(String label) {
```

Comment:
Tried to align this, and the following methods, with the existing `HasSelectByText` interface that `ComboBoxElement` implements, even though the naming is not ideal. I tried to make it more clear in the API docs that this uses item labels. Implementing `HasSelectByText` is not possible, as this only handles a single string value, so the best we can do is provide something similar for users already familiar with `ComboBoxElement`.